src/main/java/lotto/domain/LottoPrize.java

```diff
@@ -0,0 +1,41 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum LottoPrize {
+  FIRST(6, 2000000000),
+  SECOND(5, 1500000),
+  THIRD(4, 50000),
+  FOURTH(3, 5000),
+  NOTHING(0, 0);
```

Comment:
```suggestion
  FIRST(6, 2_000_000_000),
  SECOND(5, 1_500_000),
  THIRD(4, 50_000),
  FOURTH(3, 5_000),
  NOTHING(0, 0);
```
이렇게 숫자를 표현해주면 읽기가 더 수월하답니다!